examples/cc26xx/cc26xx-web-demo/httpd-simple.c

```diff
@@ -305,6 +309,30 @@ url_unescape(const char *src, size_t srclen, char *dst, size_t dstlen)
   return i == srclen;
 }
 /*---------------------------------------------------------------------------*/
+static char*
+html_escape_quotes(const char *src)
+{
+  size_t dstpos = 0;
+  memset(html_escaped_buf, 0, HTML_ESCAPED_BUFFER_SIZE);
+  for(size_t i = 0; i < HTML_ESCAPED_BUFFER_SIZE; i++) {
```

Comment:
The variable `i` is the _source_ index; but, you are comparing it to the _destination_ length.  You need to compare `dstpos` to that length.  (If there are quotation marks in the source string, then `dstpos` will be larger than `i`.  Then, a very long source string could overflow the buffer.)